src/PhpSpec/Wrapper/Subject.php

```diff
@@ -202,9 +202,9 @@ public function getFromWrappedObject(string $property)
     /**
      * @param int|string $key
      */
-    public function offsetExists($key): bool
+    public function offsetExists($key): Subject
```

Comment:
This is incompatible with the parent interface, which may become stricter. It's perhaps better to remove the type hint

Reply:
Fixed.